shim.sym

```diff
@@ -218,6 +218,14 @@
 01:d6c8 w_d6c8
 01:d6c9 w_d6c9
 01:d6ca w_d6ca
+01:d6d4 w_town_map_zone ; db
```

Comment:
If these are the levels (1-10 for Town Map, 1-4 for Safari Zone), they should say `level`, not `zone`, for consistency and to avoid confusion with the Safari Zone. (Thus `w_town_map_level` and `w_safari_map_level`, or `w_safari_zone_level`.)

Reply:
The reason I called them zones is that “level” is already used to refer to individual puzzles.

Reply:
Alternatively we could rename “levels” to “puzzles”, `level_names.asm` to `puzzle_names.asm`, etc., and repurpose “level” as you describe. I didn’t really like calling them “zones” outside of the Safari Map either.
![safari-map](https://user-images.githubusercontent.com/1767429/93917392-2ef9c100-fcc8-11ea-854b-af7176e1b42f.png)
![town-map](https://user-images.githubusercontent.com/1767429/93917406-3325de80-fcc8-11ea-953b-827b7fafcf31.png)

Reply:
Their graphics are marked "LV1", "LV2", etc, even in the Safari Zone, so yeah, `level` for those and `puzzle` for the individual puzzles.